doc/holochain_101/src/planning_a_dapp.md

```diff
@@ -1 +1,68 @@
 # Planning a dApp
+
+## Introduction
+Generally speaking, you need to know the following in order to build a holochain dApp:
+
+   how to install holochain
+   
+   how to use the command line tools
+   
+   how to configure your application with a "DNA" file
+   
+   how to write your application code in JavaScript or Lisp
```

Comment:
i'd just drop the specifics and say "how to write your application code"

Reply:
Yes, but we should not definitely not say Lisp because we currently are not even planning to provide a HDK for Lisp, even if there is a WASM backend for some Lisp..

Reply:
WAT is a Lisp 😁

Reply:
Got it, no Lisp! Provisionally going with "in a language that compiles to WebAssembly".